client/components/Groups/GroupRolesDialog.jsx

```diff
@@ -0,0 +1,77 @@
+import _ from 'lodash';
+import React, { PropTypes } from 'react';
+import { Field } from 'redux-form';
+import { Button, Modal } from 'react-bootstrap';
+import connectContainer from 'redux-static';
+import { ScopeGroup } from 'auth0-extension-ui';
+
+import createForm from '../../utils/createForm';
+
+export default createForm('groupRoles', connectContainer(class GroupRolesDialog extends React.Component {
+
+  static propTypes = {
+    group: PropTypes.object.isRequired,
+    onClose: PropTypes.func.isRequired,
+    addRoles: PropTypes.bool.isRequired
+  };
+
+  static stateToProps = (state) => {
+    const allRoles = state.roles.get('records').toJS();
+    const selectedRoles = state.groupRoles.get('records').toJS();
+    return { allRoles, selectedRoles };
+  };
+
+  renderBody(all, selected) {
+    const selectedIds = _.map(selected, role => role._id);
+    const options = _.filter(all, role => !_.includes(selectedIds, role._id)).map(role => ({
+      value: role._id,
+      text: role.name
+    }));
+
+    if (!options.length) {
+      return (
+        <Modal.Body>
+          <p className="modal-description">There are no roles you can add to this group.</p>
```

Comment:
I'm not sure if we should show this message or not. @sandrinodimattia may have an input on this.